libmscore/repeatlist.cpp

```diff
@@ -415,7 +425,7 @@ void RepeatList::unwindSection(Measure* sectionStartMeasure, Measure* sectionEnd
                   }
             if (doJump && !isGoto) {
                   Jump* s = 0;
-                  foreach(Element* e, m->el()) {
+                  foreach (Element* e, m->el()) {
```

Comment:
I've put rule in new heading "Loops" subheading "for" https://musescore.org/en/developers-handbook/musescore-coding-rules#Loops